kong/plugins/acl/handler.lua

```diff
@@ -44,16 +45,17 @@ function ACLHandler:access(conf)
   -- get the consumer/credentials
   local consumer_id = groups.get_current_consumer_id()
   if not consumer_id then
-    ngx_log(ngx_error, "[acl plugin] Cannot identify the consumer, add an ",
-                       "authentication plugin to use the ACL plugin")
-    return responses.send_HTTP_FORBIDDEN("You cannot consume this service")
+    kong.log.err("Cannot identify the consumer, add an authentication " ..
```

Comment:
It is still preferred to use the variadic form of this function, (`,` instead of `..`).